ui/src/app/pages/analysis/interactive-notebook.tsx

```diff
@@ -427,7 +427,12 @@ export const InteractiveNotebook = fp.flow(
               Preview (Read-Only)
             </div>
             {cond(
-              [!!error, () => null],
+              [
+                !!error &&
+                  (error instanceof ComputeSecuritySuspendedError ||
```

Comment:
Ah... thanks for catching and fixing this. A comment here would be good, otherwise LGTM